sheet.go

```diff
@@ -110,9 +182,15 @@ func (s *Sheet) makeXLSXSheet(refTable *RefTable, styles *xlsxStyleSheet) *xlsxW
 	maxRow := 0
 	maxCell := 0
 
+	// Scan through the sheet and see if there are any merged cells. If there
+	// are, we may need to extend the size of the sheet. There needs to be
+	// phantom cells underlying the area covered by the merged cell
```

Comment:
Except that I'm not changing the border settings, I'm literally accessing those phantom cell locations to create the underlying cells. Do you feel that the comment is confusing, or just superfluous?